### PR TITLE
CDN - intros

### PR DESCRIPTION
**Pull request by johnpapa**
added intros for heading sections

needs review before #sign-off

### CODE REVIEW

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/cdn.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-4/docs-ref-services/cdn.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180018390087-105%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/cdn.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-4/docs-ref-services/cdn.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180051078593-105%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/cdn.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-4/docs-ref-services/cdn.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180406436579-105%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/cdn.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-4/docs-ref-services/cdn.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180419050782-105%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/cdn.md [hunk 1]
```diff
@@ -13,37 +13,40 @@ ms.devlang: nodejs
...
+This example lists all of the CDN profiles.
+
 ```javascript
 const msRestAzure = require('ms-rest-azure');
 const cdnManagementClient = require('azure-arm-cdn');
 
 const subscriptionId = 'your-subscription-id';
-let cdnClient;
 
 msRestAzure.interactiveLogin((err, credentials) => {
```
Comment (jawache) [inline docs-ref-services/cdn.md]: Use promise

File: docs-ref-services/cdn.md [hunk 2]
```diff
@@ -13,37 +13,40 @@ ms.devlang: nodejs
...
-  cdnClient = new cdnManagementClient(credentials, subscriptionId);
+  const cdnClient = new cdnManagementClient(credentials, subscriptionId);
   cdnClient.profiles
     .list()
     .then(profilesList => console.log('Retrieved profiles list: ', profilesList));
```
Comment (jawache) [inline docs-ref-services/cdn.md]: Return promise and chain vs. nest to keep consistent

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/cdn.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-4/docs-ref-services/cdn.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707181347076176-105%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**